apps/examples/st_things/Kconfig

```diff
@@ -52,6 +52,13 @@ config RESET_BUTTON_PIN_NUMBER
 		The default is the reset button pin number 44 for the ARTIK 053.
 		If you use another board, you can set the pin number for the board.
 
+config CONFIG_HANDLE_GPIO_EVENT_NUMBER
```

Comment:
This has a wrong name. You should remove CONFIG_.

Reply:
Is this event number? Or GPIO number?

Reply:
this is GPIO number.

Reply:
fixed.